index.html

```diff
@@ -47,6 +47,11 @@ <h1>The Regis Altare Birthday Project 2023</h1>
         >
       </footer>
     </section>
+    <section id="card-list">
+      <!--Test image serving-->
+      <img class="card-image" src="images/elysium_delivery_services.jpg">
+      <tcg-card></tcg-card>
+    </section>
```

Comment:
move this above the footer
side note: the existing main/ section tags organization aren't quite right now that I look at it again.. I think it should be
```
<main>
  <article>
     <h1><h1>
      <section id="card-list"></section>
       <footer></footer>
  <article>
<main>
```
No need to worry about all that in your PR, I can do it in a separate PR

Reply:
Moving it now o7